.eslintrc

```diff
@@ -0,0 +1,86 @@
+{
+  "env": {
+      "node": true,
+      "mocha": true
+  },
+  "globals": {
+    "Symbol": true
+  },
+  "rules": {
+    "no-eval": 2,
+    "eqeqeq": 0,
+    "no-eq-null": 0,
+    "new-cap": 0,
+    "no-plusplus": 2,
+    "no-undef": 2,
+    "no-unused-vars": 2,
+    "brace-style": [2, "1tbs", { "allowSingleLine": true }],
+    "no-mixed-spaces-and-tabs": 2,
+    "no-multi-str": 2,
+    "quote-props": [
+      2,
+      "as-needed",
+      {
+        "keywords": true
+      }
+    ],
+    "key-spacing": 0,
+    "space-unary-ops": 0,
+    "no-spaced-func": 2,
+    "space-before-function-paren": [
+      2,
+      "never"
+    ],
+    "spaced-comment": [2, "always"],
+    "array-bracket-spacing": [
+      2,
+      "never",
+      {
+        "singleValue": false
+      }
+    ],
+    "space-in-parens": [
+      2,
+      "never"
+    ],
+    "no-trailing-spaces": 2,
+    "yoda": [
+      2,
+      "never"
+    ],
+    "comma-style": [
+      2,
+      "last"
+    ],
+    "curly": [
+      2,
+      "all"
+    ],
+    "dot-notation": 0,
+    "eol-last": 2,
+    "wrap-iife": [2, "inside"],
```

Comment:
That's causing a number of tweaks that seem to have little reason.  Were there more if it was the other way?

I'd rather do "outside", although it's a minor preference.